build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -3,12 +3,12 @@
  * Licensed under the MIT License.
  */
 
-import * as fs from "node:fs";
+import { existsSync, readFileSync, readdirSync } from "node:fs";
 import * as path from "node:path";
 import { queue } from "async";
 import * as chalk from "chalk";
 import detectIndent from "detect-indent";
-import { readFileSync, readJsonSync, writeJson, writeJsonSync } from "fs-extra";
+import { readJsonSync, writeJson, writeJsonSync } from "fs-extra";
```

Comment:
What is the difference between "fs-extra" readFileSync and node:fs version?

Reply:
The fs functions are mostly passthroughs to the node versions for the sync functions, but a major difference is that fs-extra wraps [graceful-fs](https://www.npmjs.com/package/graceful-fs), so there is some lower-level handling of some errors and whatnot. The functions that are in node:fs are all available in fs-extra, so we could align on using fs-extra everywhere. I was taking the approach of using native node functions where they exist, and fs-extra for niceties like `readJson`, but I don't have a strong preference. I just want to be consistent.

Reply:
I'm fine with either - wasn't called out as a change in the description. I saw something; said something. :)